scripts/test_cors.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash -e
+
+echo "Does the API support CORS?"
```

Comment:
to test properly we should have specific host, and check that if H match, it works if not fails.